packages/edit-navigation/src/components/menu-selector/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	MenuGroup,
+	MenuItemsChoice,
+	Button,
+	Dropdown,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import AddMenu from '../add-menu';
+
+export default function MenuSelector( { onSelectMenu, menus } ) {
+	return (
+		<div className="edit-navigation-menu-selector-wrapper">
+			<div className="edit-navigation-menu-selector-header">
+				{ __( 'Chose a menu to edit: ' ) }
```

Comment:
Typo here:
```suggestion
				{ __( 'Choose a menu to edit: ' ) }
```

I think it should be an `h3` instead of a `div`.